src/appengine/handlers/cron/ml_train.py

```diff
@@ -28,17 +28,22 @@ class Handler(base_handler.Handler):
   def get(self):
     """Handle a GET request."""
     for job in data_types.Job.query():
+
       if not utils.string_is_true(
           job.get_environment().get('USE_CORPUS_FOR_ML')):
         continue
 
+      task_name = 'gradientfuzz' if utils.string_is_true(
+          job.get_environment().get(
+              'USE_LIBFUZZER_FOR_GRADIENTFUZZ')) else 'ml_train'
```

Comment:
We're going to need to support the case where both are set for the same job. This will be especially true as we add support for the other ML jobs that are likely coming soon.

I'm also not sure I like the name of the environment variable. Could you rename this to USE_GRADIENTFUZZ? It'll be defined in a job definition, so that's clear enough when set. You'll probably also need to add a separate environment variable for something like USE_RNN, though the fact that that's how we refer to it throughout the codebase is a bit unfortunate in hindsight (and, if possible, we should rename that task to something other than ml_train to something more specific).

Reply:
Gotchu -- I'll put separate tasks for any job rolling through with `USE_CORPUS_FOR_ML`. Rename incoming!